### PR TITLE
fix: secure table load on scenario change

### PR DESCRIPTION
**Pull request by ardvi**
Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

File: src/components/ScenarioParameters/components/ScenarioParametersTabs/ScenarioParameterInput.js [hunk 1]
```diff
@@ -1,17 +1,32 @@
...
+import { useStore } from 'react-redux';
 
 const ScenarioParameterInput = ({ parameterData, parametersState, setParametersState, context }) => {
   const { t } = useTranslation();
   const parameterVarType = ConfigUtils.buildExtendedVarType(parameterData.varType, parameterData.subType);
   let varTypeFactory;
 
+  const store = useStore();
+
+  const currentScenarioId = useCallback(() => store.getState().scenario?.current?.data?.id, [store]);
```
Comment (csm-thu) [inline src/components/ScenarioParameters/components/ScenarioParametersTabs/ScenarioParameterInput.js:18]: I don't know if there is a different convention when using `useCallback`, but do you think we could rename `currentScenarioId` to `getCurrentScenarioId` to show that it is a function ?

**Review by csm-thu — APPROVED** [review]
Just one question, but LGTM :+1:

File: src/components/ScenarioParameters/components/ScenarioParametersTabs/ScenarioParameterInput.js (hunk 1 above)
Comment (ardvi) [reply src/components/ScenarioParameters/components/ScenarioParametersTabs/ScenarioParameterInput.js]: Right ;)

**Comment by ardvi** [comment]
PR merged in 315cf61c7e76db4c202f4f2350933df07f9ff7cf commit.

**Pull request closed without merging**